backend/station_manager/app/src/main/java/com/StationManager/app/storage/repository/inmemory/InMemoryRepository.java

```diff
@@ -0,0 +1,40 @@
+package com.StationManager.app.storage.repository.inmemory;
+
+import com.StationManager.app.Main;
+import com.StationManager.app.storage.repository.IRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public abstract class InMemoryRepository<T> implements IRepository<T> {
+    protected static final Logger logger = LoggerFactory.getLogger(InMemoryRepository.class);
```

Comment:
100% logger should not be here. 
Move to global static settings (`app/Settings.java`), which could later be renamed to `Context`

Reply:
I mean we only have stdout / stderr to log to. Saving serve records would be done separately and also stored in DB. So we can access it later.

Reply:
as a result of a discussion will stay with current approach. Probably will be changed in the future.